chapters/data-analysis.tex

```diff
@@ -442,18 +433,27 @@ \section{Constructing final indicators}
 
 \subsection{Integrating different data sources}
 Often, you will combine or merge information from different data sources together
-in order to create the analytical variables you are interested in.
+in order to create the analysis dataset.
 For example, you may merge administrative data with survey data
 to include demographic information in your analysis,
 or you may want to integrate geographic information
 in order to construct indicators or controls based on the location of observations.
-To do this, you will need to consider several possible linkages.
-In the simplest case, you might be merging two survey or administrative datasets
-at the same unit of response using a consistent numeric ID.
-You might also be combining datasets with the same unit of response,
-but that use an external or string ID such as a name,
-which do not always perfectly match.
-In these cases, you will need to extensively analyse the merging patterns
+To do this, you will need to consider the unit of response for each dataset, 
+and the identifying variable, to understand how they can be merged.
+
+If the datasets you need to join have the same unit of response, 
+merging may be straightforward.
+The simplest case is merging datasets at the same unit of response 
+which use a consistent numeric ID.
```

Comment:
```suggestion
which use a consistent, uniquely and fully identifying ID variable. 
```
We have link to what _uniquely and fully identifying_ means earlier in this chapter